### DISCUSSION TITLE
OpenAI API pending and timeout error?

### DISCUSSION THREAD

**Discussion by kientv** [FlowiseAI/Flowise#1256] [Q&A]
Is this problem of OpenAI or Flowise?

![image](https://github.com/FlowiseAI/Flowise/assets/3217438/06985abe-59fd-4b61-91bb-90c02706b6b3)

![image](https://github.com/FlowiseAI/Flowise/assets/3217438/012d86c3-db0e-46f7-a826-a4e2469a5600)

**Comment by kientv** [comment]
> chatbot  | 2023-11-20 14:28:35 [INFO]: ⬆️ POST /api/v1/prediction/277241d0-8e2b-4fc3-bdb0-bc89123489c1
chatbot  | Created a chunk of size 1073, +
chatbot  | which is longer than the specified 1000
chatbot  | 2023-11-20 14:28:36 [INFO]: [server]: Chatflow 277241d0-8e2b-4fc3-bdb0-bc89123489c1 added into ChatflowPool
chatbot  | 2023-11-20 14:30:38 [ERROR]: [server]: Error: Request timed out.
chatbot  | TimeoutError: Request timed out.
chatbot  |     at wrapOpenAIClientError (/usr/src/packages/node_modules/langchain/dist/util/openai.cjs:9:17)
chatbot  |     at /usr/src/packages/node_modules/langchain/dist/chat_models/openai.cjs:521:69
chatbot  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
chatbot  |     at async RetryOperation._fn (/usr/src/packages/node_modules/p-retry/index.js:50:12)

**Comment by HenryHengZJ** [comment]
pretty sure its on OpenAI side
